website/docs/cloud-docs/vcs/bitbucket-server.mdx

```diff
@@ -91,7 +91,9 @@ Leave the page open in a browser tab, and remain logged in as an admin user.
 
    ![Bitbucket Server screenshot: The application links page](/img/docs/bitbucket-server-application-links.png)
 
-1. Enter Terraform Cloud's URL in the text field (`https://app.terraform.io`, or the hostname of your Terraform Enterprise instance) and click the "Create new link" button.
+1. In application links, select Create link.
+2. Select Atlassian product as the link type. This option also works for external applications. This is the original application links mechanism – Atlassian has left it under this option so users can still use their OAuth 1.0 integrations.
```

Comment:
I suggest rewording to "This method is the original application links mechanism. Atlassian has left it under this option so users can still use their OAuth 1.0 integrations.